sql/moz-fx-data-shared-prod/org_mozilla_firefox/event_types/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
```

Comment:
I would love for _every_ generated file to include a comment that indicates it's been generated by a specific script, including all .sql variants and the metadata.yaml.